fe/src/main/java/org/apache/doris/analysis/ShowPartitionsStmt.java

```diff
@@ -73,17 +107,7 @@ public ProcNodeInterface getNode() {
 
     @Override
     public void analyze(Analyzer analyzer) throws AnalysisException, UserException {
-        super.analyze(analyzer);
-
-        if (Strings.isNullOrEmpty(dbName)) {
-            dbName = analyzer.getDefaultDb();
-            if (Strings.isNullOrEmpty(dbName)) {
-                throw new AnalysisException("No db name in show data statement.");
-            }
-        } else {
-            dbName = ClusterNamespace.getFullName(getClusterName(), dbName);
-        }
-
+        analyzeSynTax(analyzer);
```

Comment:
`analyze` itself means `analyze syntax`, so no need to name like `analyzeSynTax`.
`analyzeImpl` is better. Like named in other class.